admin/class-surveyfunnel-lite-admin.php

```diff
@@ -910,25 +910,37 @@ public function surveyfunnel_lite_register_gutenberg_blocks() {
 	 * @param array $attributes survey attributes.
 	 */
 	public function surveyfunnel_lite_gutenberg_display_single_survey( $attributes ) {
-		$survey_atts       = array();
+		$survey_atts = array();
+		$shortcode   = '[surveyfunnel_lite_survey';
+
 		$survey_atts['id'] = 0;
 		if ( isset( $attributes['survey_id'] ) ) {
 			$survey_atts['id'] = $attributes['survey_id'];
 		}
+		$shortcode          .= ' id="' . $survey_atts['id'] . '"';
 		$survey_atts['type'] = 'responsive';
```

Comment:
instead of adding values to array, it can initialised with default values at the beginning on line no 913.
For example: $survey_atts = array('id'=> 0, 'type'=> 'responsive',...)